CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+- Verbal or physical violence, threats, intimidation, bullying, etc. This includes subtle belittlement message or unwelcoming language. No matter the intent, such behaviour can have a negative impact on others. 
+
+This Code of Conduct is not intended to be exhaustive or to detail all forms of unacceptable behaviours. It is not designed to rank the behaviours or to consider that some behaviours do not deserve to be punished. 
+
+If a member feels harassed or in an uncomfortable situation, they will be asked to report the situation according to the procedure described hereafter. 
+
+Every report will be examined. 
+
+
+## Reporting issues
+
+If you experience or witness misconduct, you may: 
+- Flag it thanks to the “report content” button on the contributory sites; 
+- Address it directly, and let the person know how this behaviour affects you or others;
+- If the situation is an online event, you may flag the content to the following address: [https://www.prestashop.com/en/abuse](https://www.prestashop.com/en/abuse);
+- If the situation happens during a physical event, you may address it directly to a member of the organization or a security officer. 
+
+Your report must include the followings: 
+- Your contact information, where you can be reached. Anonymous report cannot be treated; 
+- Your name and the name of the person or the people involved;
+- A description of what happened, and if you believe the situation is ongoing; don’t forget to provide the link of the webpage or the discussion; 
+- Any additional information that may be helpful.
+
+After filing a report, the most adapted person will handle the situation in order to check it. This person may ask you additional questions and will determine, with you, how to respond and handle the situation. 
+
+If the person harassing you is a moderator, they will be asked to recuse themselves. If the report is filed by a moderator, another moderator will handle the situation. 
+
+
+## Sanction 
+
+Each situation will be handled on a case-by-case basis. 
+
+In the event of a non-compliance with the principles in the Code of Conduct, the warning procedure shall be initiated: 
+- On the first occurrence, the moderator will send a public or private warning, and the content may be modified or deleted, 
+- According to the situation, the account of the person may be temporarily suspended, 
+- For certain cases, such as extreme behaviours that affect the community, threats, acts of violence, a ban of the community may be applied along with a ban to participate in an event. 
+
+In case of participation in an event, a member of the organization will contact the alleged offender directly to explain the facts alleged and to inform him of the actions that may be taken against him. 
+
+In the event of repetitive misconduct, the member of the organization reserves the right to exclude a participant. 
+
+If you have any doubts about how the situation has been handled, you can contact PrestaShop via this
```

Comment:
> you can contact PrestaShop

Do you mean the PrestaShop Company? Or PrestaShop maintainers? Or the PrestaShop Company Staff?

Reply:
that's a branding issue and we can not solve it in this document. the link to the form makes it clear that it's "PrestaShop" in a global way.
then it will be sorted and sent to the best folks.